src/App.tsx

```diff
@@ -17,18 +59,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                searchQuery={searchQuery}
+                setSearchQuery={setSearchQuery}
+                isCompleted={isCompleted}
+                setIsCompleted={setIsCompleted}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading
+                ? (
+                  <TodoList
+                    todos={visibleTodos}
+                    selectedTodo={selectedTodo}
+                    onSelect={selectTodo}
+                  />
+                )
+                : <Loader />}
```

Comment:
if is loading - we render todolist, and if not - we render loader? Does this make sense?